content/strategy-goals/strategy/code-graph/index.md

```diff
@@ -1,30 +1,28 @@
+Code Graph comprises teams working on [Search](./search/index.md), [Code Insights](./code-insights/index.md), [Code Intelligence](./code-intelligence/index.md), and [Batch Changes](./batch-changes/index.md); this page describes the mission and strategy that ties them together over the next year or so, with each linked page providing a more detailed strategy for that area. There is also a whole-company [Sourcegraph strategy](../index.md) and [Product & Engineering strategy](../../../departments/product-engineering/strategy-goals/index.md) to which all of this is aligned. Our [roadmap tracker](https://github.com/orgs/sourcegraph/projects/214/views/34) (internal only) shows what we're delivering to make this strategy a reality.
 
-If you're new to our strategy pages, this video walkthrough will help orient you.
+If you're new to our strategy pages, this video walkthrough (last updated Feb 2022) will help orient you.
 
 <div style="position: relative; padding-bottom: 56.25%; height: 0;"><iframe src="https://www.loom.com/embed/71b675b67c464b0db7670a7f5025cbbe" frameborder="0" webkitallowfullscreen mozallowfullscreen allowfullscreen style="position: absolute; top: 0; left: 0; width: 100%; height: 100%;"></iframe></div>
 
-## Themes
-
-Our mission is to **make discovering, learning, and editing code easier, no matter how complex it is, where it's stored, or how experienced you are.** We will achieve this through several important themes:
-
-### Make the power of our features easier to find and use
+## Mission
 
-Getting the most out of our product can take time as you get up to speed with what's possible. Making it easier to learn will help teams find the value more quickly than today. Our task is to find a way to make interacting with multiple languages and repositories easy to understand.
+Our mission is to **make discovering, learning, and editing code easier, no matter how complex it is, where it's stored, or how experienced you are.**
 
-### Level up our enterprise-ready features
+## Themes
 
-How pricing, upgrade paths, auditing, single sign-on, access control, etc. work are essential to our customers with complex organizations. In partnership with our Cloud team, we're focused on ensuring we exceed their needs so that rolling out Sourcegraph is easy to do.
+Apart from the work each of our teams do for our mission, across Code Graph we're particularly focused now on improving the **discoverability** and **ease-of-use** around our [use cases](../index.md#use-cases). There are a few reasons why this is important:
 
-### Enable our use cases
+- Focusing on use cases ensures we're prioritizing areas where people get real value out of the product.
+- Improving discoverability reduces the chance of people giving up on our features because they are too complicated/not intuitive
```

Comment:
```suggestion
- Improving discoverability increases feature adoption because it brings users into features at moments when the features would be most useful
```
I am in favor of reducing complexity/onboarding learning features etc but that felt to me like it's not directly related to discoverability, and I also felt like phrasing it in the positive case might be clearer here. (Feel free to ignore this.)

Reply:
Positive case is indeed better, thanks for the suggestion this is a great improvement!